ui/selectors/multichain.ts

```diff
@@ -30,7 +30,20 @@ import {
 
 export type MultichainState = AccountsState & {
   metamask: {
-    // TODO: Use states from new {Rates,Balances,Chain}Controller
+    balances: {
```

Comment:
I wonder if we could use the same pattern than for `AccountsState` and use something like:

```ts
export type MultichainState = {
  metamask: BalancesControllerState & RatesControllerState & AccountsControllerState,
};
```

WDYT?

Reply:
Yea, i think in the long term we should fix the `CombinedBackgroundAndReduxState` to have it be properly typed. We can do this in another PR